### PR TITLE
Cleanup some memory flags used in the CLI

### PR DESCRIPTION
**Pull request by dnephin**
Remove all the duplicate code around memory CLI flags

### CODE REVIEW

**Comment by vieux** [comment]
LGTM!

File: opts/opts.go [hunk 1]
```diff
@@ -444,3 +444,39 @@ func (m *MemBytes) UnmarshalJSON(s []byte) error {
 	*m = MemBytes(val)
 	return err
 }
+
+// MemSwapBytes is a type for human readable memory bytes (like 128M, 2g, etc).
+// It differs from MemBytes in that -1 is valid and the default.
+type MemSwapBytes int64
+
+// Set sets the value of the MemBytes by passing a string
```
Comment (allencloud) [inline opts/opts.go]: MemBytes  -> MemSwapBytes ?

**Review by vdemeester — APPROVED** [review]
LGTM 🐸

**Comment by dnephin** [comment]
Fixed the comment, thanks!

**Pull request merged**